src/jade/resources/default_cfg/benchmarks_pp/raw/openmc/FNG-HCPB.yaml

```
@@ -0,0 +1,140 @@
+  16023: 28.402
+  16027: 28.403
+  16031: 28.404
+  16035: 28.405
+  16039: 28.406
+  16043: 28.407
+  16047: 28.408
+  16051: 28.409
+  16055: 28.410
+  16059: 28.411
+  16063: 28.412
+_cell_id_h3: &depth_h3d
```

YAMLlint (1.37.1)

[error] duplication of key "_cell_id_h3" in mapping

(key-duplicates)